knexfile.js

```diff
@@ -0,0 +1,32 @@
+module.exports = {
+  development: {
+    client: 'sqlite3',
```

Comment:
Awesome job on the setting up the knexfile.js, it would be nice if we could have a uniform database, like using postgres for all the DB environments